build.analyze.sh

```diff
@@ -74,9 +74,14 @@ fi
 mkdir -p $BUILD_DIR
 mkdir -p $REPORT_DIR
 rm -Rf $BUILD_DIR/*
+rm -Rf $REPORT_DIR/*
```

Comment:
Not necessary as `$REPORT_DIR` is relative to `BUILD_DIR`. In fact, the mkdir statement is wrong: it should be `mkdir -p $BUILD_DIR/$REPORT_DIR`